static/app/views/organizationStats/usageStatsOrg.tsx

```diff
@@ -39,17 +41,26 @@ type Props = {
     pagePeriod?: string | null;
     transform?: ChartDataTransform;
   }) => void;
+  isSingleProject: boolean;
   organization: Organization;
   projectIds: number[];
   chartTransform?: string;
 } & AsyncComponent['props'];
 
-type State = {
+type UsageStatsOrganizationState = {
   orgStats: UsageSeries | undefined;
 } & AsyncComponent['state'];
 
-class UsageStatsOrganization extends AsyncComponent<Props, State> {
-  componentDidUpdate(prevProps: Props) {
+/**
+ * This component is replaced by EnhancedUsageStatsOrganization in getsentry, which inherits
+ * heavily from this one. Take care if changing any existing function signatures to ensure backwards
+ * compatibility.
+ */
+class UsageStatsOrganization<
```

Comment:
This is the reason behind a lot more `get method()s` in this component.

By removing the props work from the render work, we can use the component in getSentry to modify the props while maintaining the render method or vice versa. For example, using `this.chartProps`, appending new data, and then leaving `renderChart` to work as written in Sentry.